packages/hardware-trezor/src/transformers/tx.ts

```diff
@@ -14,14 +14,21 @@ import { mapAuxiliaryData, mapCerts, mapTxIns, mapTxOuts, mapWithdrawals } from
 const trezorTxTransformer = async (
   body: Cardano.TxBody,
   context: TrezorTxTransformerContext
-): Promise<Partial<Trezor.CardanoSignTransaction>> => ({
-  auxiliaryData: body.auxiliaryDataHash ? mapAuxiliaryData(body.auxiliaryDataHash) : undefined,
-  certificates: mapCerts(body.certificates ?? [], context),
-  fee: body.fee.toString(),
-  inputs: await mapTxIns(body.inputs, context),
-  outputs: await mapTxOuts(body.outputs, context),
-  withdrawals: mapWithdrawals(body.withdrawals ?? [], context)
-});
+): Promise<Partial<Trezor.CardanoSignTransaction>> => {
+  const certificates = mapCerts(body.certificates ?? [], context);
```

Comment:
Is there a particular reason you isolated certificates?

Reply:
Ah thanks, that's a leftover 👍 will refactor to simpler version

Reply:
@tomislavhoracek fixed in latest commit